.github/workflows/example.yml

```
@@ -41,14 +48,17 @@ jobs:
         uses: ./
         with:
           working-directory: example
-          stack-arguments: --resolver ${{ matrix.resolver }}
-          cache-prefix: ${{ matrix.resolver }}/
-
-      - if: ${{ runner.os != 'windows' }}
+          stack-arguments: --resolver ${{ matrix.stack.resolver }}
+          cache-prefix: ${{ matrix.stack.resolver }}/
+      
+      - if: matrix.stack.resolver != 'lts' && matrix.stack.resolver != 'nightly'
```

I left a comment on the commit itself, but GitHub had a warning so maybe you rewrote history already, so I'll repeat it here:

Wouldn't,

```suggestion
      - if: matrix.ghc
```

Be more directly what you mean?

---

Well, yes. It's weird that `truthy` isn't [documented](https://docs.github.com/en/actions/learn-github-actions/expressions#example) here. Seems like they use the JS [definition](https://developer.mozilla.org/en-US/docs/Glossary/Truthy).